common/prove/deposit.go

```diff
@@ -37,8 +38,8 @@ func (w *WitnessHelper) constructDepositTxWitness(cryptoTx *TxWitness, oTx *Tx)
 	return cryptoTx, nil
 }
 
-func toCryptoDepositTx(txInfo *legendTxTypes.DepositTxInfo) (info *CryptoDepositTx, err error) {
-	info = &CryptoDepositTx{
+func toCryptoDepositTx(txInfo *legendTxTypes.DepositTxInfo) (info *std.DepositTx, err error) {
+	info = &std.DepositTx{
```

Comment:
Question: why the package `github.com/bnb-chain/zkbnb-crypto/legend/circuit/bn254/std` is named as `std`?